docs/src/pages/docs/installation/kubernetes.mdx

```diff
@@ -94,25 +94,18 @@ postgresql:
 
 #### Dependencies
 
-You can specify pip packages to be installed before startup, e.g. to install extra database drivers:
+You can specify pip packages to be installed before startup. The following example installs the Big Query and Elasticsearch database drivers so that you can connect to those datasources in your Superset installation.
```

Comment:
Let's include the comment on the values config file:
```
# Install additional packages and do any other bootstrap configuration in this script
# For production clusters it's recommended to build own image with this step done in CI
```
Since we should not encourage the use of a bootstrap script to install OS or python packages

Reply:
updated.